### DISCUSSION TITLE
Glow Hologram

### DISCUSSION THREAD

**Discussion by Timofffee** [Minestom/Minestom#1631] [Help]
Hello. I saw that there are plugins for Spigot that allow the text in the hologram to glow regardless of the lighting around. How can this be repeated?

For example: https://www.spigotmc.org/resources/hologramglow.61737/

**Comment by Argedon** [comment]
U need to set entity on fire with EntityMeta.setOnFire(boolean).

**Reply by Timofffee** [reply]
It works, but not the way I need it. The text really gets brighter, but at the same time it starts to emit light and illuminate everything around, which does not look the way I would like.

**Reply by Argedon** [reply]
In that case, you should modify the EntityMetaDataPacket, save it as a variable and send it every time the player sees the ArmorStand. With that server shouldn't create any light. For example, you can achieve it like that:

```java
public class Hologram extends EntityCreature {
    private final EntityMetaDataPacket metaDataPacket;

    public Hologram(String text) {
        super(EntityType.ARMOR_STAND);

        ArmorStandMeta armorMetadata = (ArmorStandMeta) getEntityMeta();
        armorMetadata.setNotifyAboutChanges(false);
        armorMetadata.setMarker(true);
        armorMetadata.setInvisible(true);
        armorMetadata.setOnFire(true);
        armorMetadata.setCustomNameVisible(true);
        armorMetadata.setCustomName(Component.text(text));
        metaDataPacket = super.getMetadataPacket();
        armorMetadata.setOnFire(false);
        armorMetadata.setNotifyAboutChanges(true);
    }

    @Override
    public @NotNull EntityMetaDataPacket getMetadataPacket() {
        return metaDataPacket;
    }
}
```

```java
Hologram hologram = new Hologram("test");
hologram.setInstance(player.getInstance(), player.getPosition());
 ```

**Reply by Timofffee** [reply]
I'm probably doing something wrong.
![image](https://user-images.githubusercontent.com/5556081/209906482-821f67ed-68ee-45f2-a39a-b60e6c117520.png)
